### PR TITLE
onr last change

### PR DESCRIPTION
**Pull request by sidarun009**
please suggestion sir?

### CODE REVIEW

File: Hello.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 console.log("hello");
 console.log("sidhu");
-console.log("sidhu added another line for review");
\ No newline at end of file
+console.log("sidhu added another line for review");
+console.log("one last");
```
Comment (shaikafroz786) [inline Hello.js:4]: adding bekar ka line,please remvove this

**Review by shaikafroz786 — COMMENTED** [review]
oerall fine tho

File: Hello.js (hunk 1 above)
Comment (sidarun009) [reply Hello.js:4]: okie

Comment (shaikafroz786) [reply Hello.js:4]: remove last line

**Pull request merged**